### ISSUE TITLE
Support opensuse leap 15.1

### ISSUE THREAD

**Issue by hsbt** [itamae-kitchen/mitamae#99]
AWS provides the opensuse leap 15.1 currently.

https://aws.amazon.com/marketplace/pp/Cabelo-openSUSE-Leap-151/B07S6NP2SM

The latest version of mitamae is not working with opensuse leap 15.1. So, `node[:platform]` returns empty string. But `node[:platform_version]` returns `15.1` correctly. I'm not sure why `platform` was missed with the opensuse leap 15.1.

**Comment by hsbt** [comment]
`os-release` is here

```
ec2-user@ip-172-31-3-159:~> cat /etc/os-release
NAME="openSUSE Leap"
VERSION="15.1"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.1"
PRETTY_NAME="openSUSE Leap 15.1"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.1"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```

**Comment by k0kubun** [comment]
fixed in v1.11.4
